src/schema/ecommerce/types/order.ts

```diff
@@ -181,7 +192,10 @@ export const {
   connectionType: OrderConnection,
   edgeType: OrderEdge,
 } = connectionDefinitions({
-  nodeType: OrderType,
+  // TODO: DO NOT MERGE WITHOUT VERIFYING THIS
+  // and maybe updating @types/graphql
+  // @ts-ignore
+  nodeType: OrderInterface,
```

Comment:
The d.ts says a nodeType can only be a graphql object type, but it can also be an interface, this'll need adding to the types - @ashkan18 did you test whether the connection was fine?

Reply:
not yet, need to try that and it's harder to setup than what you think :)

Reply:
alright! confirmed! connections work! ✅ 

![image](https://user-images.githubusercontent.com/1230819/48864228-eaceca00-ed99-11e8-9715-99b08b56e83f.png)

Reply:
I've looked into this from the types perspective, this is the corresponding issue - https://github.com/graphql/graphql-relay-js/issues/41

In *2015* relay could not support interface connections - but maybe relay modern can

Reply:
The relay compiler was fine with it when we migrated the schema in, so I think it's likely that that changed